judo-psm-generator-jaxrs-test/pom.xml

```diff
@@ -296,13 +327,38 @@
                                 <source>${basedir}/target/generated-sources/psm/jaxrs/ParameterizedQueryTest</source>
                                 <source>${basedir}/target/generated-sources/psm/jaxrs/FormsTest</source>
                                 <source>${basedir}/target/generated-sources/psm/jaxrs/CardsAndLabelsTest</source>
-                                <source>${basedir}/target/generated-sources/psm/jaxrs/ActionGroupTest</source>
+                                <source>${basedir}/target/generated-sources/psm/jaxrs/AuthenticationTest</source>
                                 <source>${basedir}/target/generated-sources/psm/jaxrs/ApplicationSelectionTest</source>
+                                <source>${basedir}/target/generated-sources/psm/jaxrs/ActionGroupTest</source>
                             </sources>
                         </configuration>
                     </execution>
                 </executions>
             </plugin>
+            <!--
+            <plugin>
+                <groupId>io.swagger.core.v3</groupId>
+                <artifactId>swagger-maven-plugin</artifactId>
+                <version>2.2.19</version>
+                <executions>
+                    <execution>
+                        <configuration>
+                            <outputFileName>openapi</outputFileName>
+                            <outputPath>${project.build.directory}/generatedtest</outputPath>
+                            <outputFormat>JSONANDYAML</outputFormat>
+                            <resourcePackages>
+                                <package>hu.blackbelt.judo.psm.generator.jaxrs.test</package>
+                            </resourcePackages>
+                            <prettyPrint>TRUE</prettyPrint>
+                        </configuration>
+                        <phase>compile</phase>
+                        <goals>
+                            <goal>resolve</goal>
+                        </goals>
+                    </execution>
+                </executions>
+            </plugin>
+            -->
```

Comment:
Will this commented code be used in the future?

Reply:
If you want to make changes later, you just have to remove the comment and you can test it.